src/cli/commands/help.js

```diff
@@ -14,45 +14,50 @@ export function hasWrapper(flags: Object, args: Array<string>): boolean {
 
 export function setFlags(commander: Object) {}
 
+function rawText(str: string): {inspect(): string} {
+  return {
+    inspect(): string {
+      return str;
+    },
+  };
+}
```

Comment:
It is but the alternative is to fix the `JSON.stringify` transformation that the reporter uses on strings, which requires a `typeof x === 'string'` check and then fixing all translatable strings to add the now-missing quotes which would invalidate all translations. If you think it's still worth it, I'm okay doing that in a follow-up since this stringification issue popped up in a few other PRs recently.